pkgs/by-name/re/reposilite/package.nix

```
@@ -1,11 +1,20 @@
 {
-  stdenv,
   lib,
+  stdenv,
   fetchurl,
-  makeWrapper,
   jre_headless,
+  linkFarm,
+  makeWrapper,
+  plugins ? [ ],
 }:
-
+let
+  pluginsDir = linkFarm "reposilite-plugins" (
+    builtins.map (p: {
+      name = (builtins.parseDrvName p.name).name + ".jar";
+      path = p;
```

```suggestion
      path = p.outPath or p;
```
In case the plugin is a derivation